win32/win_opendir.h

```diff
@@ -18,6 +18,10 @@
 
 #include <io.h>
 
+#if defined(_MSC_VER) && _MSC_VER < 1800
+#include "stdint.h" // intptr_t
```

Comment:
Does this always prefer the local copy? Or do we need something like `"./stdint.h"` or maybe even `"../win32/stdint.h"` for consistency reasons?

Reply:
I don't know, it seems to work in VC6 and VC2008.